docs/json_rpc_api.md

```diff
@@ -566,6 +567,18 @@ Example Return|`{`<br />&nbsp;&nbsp;`"bytes": 310768,`<br />&nbsp;&nbsp;`"size":
 |Example Return|`true`|
 [Return to Overview](#MethodOverview)<br />
 
+***
+<a name="version"/>
+
+|   |   |
+|---|---|
+|Method|version|
+|Parameters|None|
+|Description|Returns the version of the JSON-RPC API built into this release of btcd.|
+|Returns|string|
+|Example Return|1.0.0|
```

Comment:
Still missing the key for "btcdjsonrpcapi".  The idea with this object format is that we can extend it to provide versions of other systems or programs (for example, when wallet adds this, it can report both api versions, because that matters to clients who are using rpc passthrough).